frontend/src/buttons/LaunchButton/LaunchButton.tsx

```diff
@@ -42,16 +51,29 @@ export const LaunchButton: React.FC<Props> = ({ connection, service }) => {
         ...connection,
         username: username.toString(),
       })
-    window.open(app.launch({ ...connection, username }))
+    const launchApp = app.launch({ ...connection, username })
+
+    app.launchBrowser(app.title) && false ? window.open(launchApp) : emit('service/launch', launchApp)
```

Comment:
```suggestion
    app.launchBrowser(app.title) ? window.open(launchApp) : emit('service/launch', launchApp)
```
Looks like debug code here